build.gradle

```
@@ -186,6 +186,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named("githubRelease") {
```

```suggestion
tasks.named("githubRelease").configure {
```
and for the others too to make it use the task configuration avoidance api

---

@runningcode on this one, I was not sure if it is needed to get lazy task instantiation. It's defintely needed?

---

If so, we would also have to do it for `publishPlugins` (and on the other 3 repos where we have PRs open).

---

I think `tasks.named` is ok to use according to the [task configuration avoidance guide](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html).

That being said, it is currently [being eagerly configured](https://ge.solutions-team.gradle.com/s/2ijgwbnylh2cy/performance/configuration?details=code-unit-net.breadmoirai.github-release!:&openScriptsAndPlugins=WyJjb2RlLXVuaXQtbmV0LmJyZWFkbW9pcmFpLmdpdGh1Yi1yZWxlYXNlIl0) which is why I brought it up. I think that is an issue in the plugin itself and not this buildscript.

Conclusion: we can leave out `.configure` here.

---

Got it. I guess it does not hurt to have it here.